From 9830eecd6989198882fa2c4ef132665f36ee7abe Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 31 Jul 2013 08:06:56 +0200
Subject: disable s3/s4 by default

Message-id: <1374235471-27096-2-git-send-email-armbru@redhat.com>
Patchwork-id: 52604
O-Subject: [RHEL-7 PATCH qemu-kvm 1/2] disable s3/s4 by default
Bugzilla: 980840
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Gleb Natapov <gleb@redhat.com>

From: Gleb Natapov <gleb@redhat.com>

After the changes proposed in Bug 827503, S3/S4 will be enabled by
default on qemu/seabios, but we want it disabled by default in RHEL6.4
to avoid problems on problematic OSes (it'll be enabled on a
case-by-case basis in the management layer).

Since we want to have it fully supported when enabled, QE should perform
the expected tests (which AFAIK have been run in rhel6.3 already, so
enabling it manually should not be a problem).

Signed-off-by: Gleb Natapov <gleb@redhat.com>
Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
(cherry picked from commit 625992f4673989108819db751aa9066b65fc6020)

Picked from qemu-kvm-rhel6.

Conflicts:
        hw/acpi.c
Signed-off-by: Markus Armbruster <armbru@redhat.com>

diff --git a/hw/acpi/piix4.c b/hw/acpi/piix4.c
index c4af1cc..86ff598 100644
--- a/hw/acpi/piix4.c
+++ b/hw/acpi/piix4.c
@@ -475,8 +475,8 @@ i2c_bus *piix4_pm_init(PCIBus *bus, int devfn, uint32_t smb_io_base,
 
 static Property piix4_pm_properties[] = {
     DEFINE_PROP_UINT32("smb_io_base", PIIX4PMState, smb_io_base, 0),
-    DEFINE_PROP_UINT8("disable_s3", PIIX4PMState, disable_s3, 0),
-    DEFINE_PROP_UINT8("disable_s4", PIIX4PMState, disable_s4, 0),
+    DEFINE_PROP_UINT8("disable_s3", PIIX4PMState, disable_s3, 1),
+    DEFINE_PROP_UINT8("disable_s4", PIIX4PMState, disable_s4, 1),
     DEFINE_PROP_UINT8("s4_val", PIIX4PMState, s4_val, 2),
     DEFINE_PROP_END_OF_LIST(),
 };
